### DISCUSSION TITLE
为什么打开MP4打开不了，而且PDF也看不了，新手小白求问

### DISCUSSION THREAD

**Discussion by Xingyank9** [AlistGo/alist#6411] [Q&A]
![16ab2237795339d265ff311a21f4760](https://github.com/alist-org/alist/assets/85680814/af3df71c-59fa-4664-baad-cbdc25952e4c)

**Comment by pongfcnkl** [comment]
mp4检查浏览器是否支持当前编码
pdf可以搭建OnlyOffice

**Comment by bestWuTong** [comment]
依个人经历来看，直接**重启Alist服务**或许能解决
